packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -316,7 +316,7 @@ class Dropdown extends React.Component {
                                         <ThemedScrollbars
                                             autoHeight
                                             autoHide
-                                            autoHeightMax={404}
+                                            autoHeightMax={220} // As specified by design spec
```

Comment:
can we pass this in? the drop-downs in cashier should be 404 height

Reply:
`404` was causing issues with double scrollbars whenever a mouse is connected in other dropdowns with long lists. Maybe we should pass autoHeightMax as a prop instead.